### PR TITLE
Add slack activity to release report.

### LINKED ISSUE #1367: Contributor reports based on slack interaction
since a lot of mailing list activity has moved to slack, we are considering options to create reports based on slack interaction through the API.

This is in its beginning stages, so we are looking for ideas and feedback. 
Consider looking at this part of the api for what kind of data may be available to use:
[https://api.slack.com/admins/audit-logs](https://api.slack.com/admins/audit-logs)

### PR DESCRIPTION
**Pull request by brianjp93**
- fixes #1367

### CODE REVIEW

**Comment by brianjp93** [comment]
image

**Comment by brianjp93** [comment]
I'll leave it up to @rbbeeston to present the data however you'd like.  I just made it mirror what was being shown for the mailinglist data for now

**Review by rbbeeston — COMMENTED** [review]
The data itself looks good, I'm undecided on the wording, but this can be fixed later.

File: libraries/forms.py [hunk 1]
```diff
@@ -608,6 +609,41 @@ def apply_colors(self):
         graph.apply_colors()
         return graph
 
+    def _get_slack_stats(self, prior_version, version):
+        """Returns all slack related stats."""
+        start = prior_version.release_date
```
Comment (rbbeeston) [inline libraries/forms.py:614]: @vinniefalco Should we have a way to filter to specific channels, or are we always going to use all channels?

Comment (brianjp93) [reply libraries/forms.py:614]: As it is, it is aggregating stats for all channels the bot as access to (which is just general and boost-website atm I believe).

Comment (vinniefalco) [reply libraries/forms.py:614]: only channels starting with "boost"

**Comment by vinniefalco** [comment]
I think we should show activity by channel, and only counting channels starting with "boost". And sorted by activity

**Comment by brianjp93** [comment]
Example with 2 channels on a page, I only have the boost-website channel so I just duplicated it for this example.
image

**Pull request merged**